packages/sitecore-jss-react/src/enhancers/withPlaceholder.test.tsx

```diff
@@ -193,4 +200,29 @@ describe('withPlaceholder HOC', () => {
       });
     });
   });
+
+  testDataWithoutParams.forEach((dataSet) => {
+    describe(`with ${dataSet.label}`, () => {
+      it('should render a placeholder with given key when params are not passed', () => {
+        const component = (dataSet.data.sitecore.route.placeholders.main as (
+          | ComponentRendering
+          | RouteData
+        )[]).find((c) => (c as ComponentRendering).componentName);
+        const phKey = 'page-content';
+        const props: PlaceholderProps = {
+          name: phKey,
+          rendering: component,
+          params: { test: 'test' },
```

Comment:
params are passed here for this test. Is this an omission?

Reply:
Ohh, good catch.
This got left out when I was doing initial testing. 
Removed :)